mail-client/mutt-wizard/Manifest

```diff
@@ -0,0 +1 @@
+DIST mutt-wizard-20200430.tar.gz 32140 BLAKE2B 37656f1a25eecf5caa982dc898e0c3fc45b38b1c3d2db8673ecdf1a3694d3c3a856239830bdc86182b97e84b4c18d7ed42720be4ec60125cc5060817df6ff262 SHA512 aa653421a3492c19a88e0ed421f554d7e0af26d58c7420887b0be2b9db6f481cd439da359af8207fd11fb7180b289535038ad0ce94700c8123c84445aa83ed90
```

Comment:
Now I don't know if no one spotted this already or if you changed it, but we name packages like these `mutt-wizard-0_pre20200430.ebuild`. Then when whatever release is made, it will always be an update. Now 20200430 -> 1.0 would be a downgrade.

Rest looks good!

Reply:
lmao, wow, I didn't even know that.  
I didn't change it, will fix it :smile_cat:

Reply:
done